gwtlib/branches.py

```
@@ -1,19 +1,8 @@
 # gwtlib/branches.py
-from gwtlib.git_ops import run_git_command, run_git_quiet
-
+import subprocess
+from typing import Optional, Tuple
```

Ruff (0.15.10)

[warning] `typing.Tuple` is deprecated, use `tuple` instead

(UP035)